src/schemas/evaluation_schema.py

```diff
@@ -220,6 +220,10 @@
                 "description": "Whether to print the table explaining question-wise verdicts",
                 "type": "boolean",
             },
+            "should_export_explanation_csv": {
```

Comment:
can you also add a default value for this in `src/schemas/defaults/evaluation.py`?

Reply:
Also please update one of the sample files to generate this output for the user who runs all samples during onboarding.